### PR TITLE
857 add talent to existing contribution

### LINKED ISSUE #857: [BUG] Adding Talent to existing Contribution is not working
**Describe the bug**
When the user adds a new Talent to an existing Contribution containing a Capability, the form elements are not loading.

**To Reproduce**
1. Add a Contribution with a Capability
2. Edit Contribution and add a Talent
3. The form elements are not appearing as shown in the screenshot below.

**Expected behavior**
The form elements for the new Talent should load properly.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: MAcOS - Big Sur
 - Browser: Chrome
 - Version: develop branch

**Additional context**

### PR DESCRIPTION
**Pull request by minump**
## Description
Code changes are made to include post.talent requests and then check if the form is editable. This way even if the form is not editable, it will be displayed.

**Fixes #857**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Hi, @minump, I tested with the steps to repeat described in the Github issue and this seems to be fixed. Please fix the merge conflicts. I am approving this. Thanks.

Hi, @ywkim312, do you have any comments on this PR? Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Comment by minump** [comment]
Converting to draft PR . Awaiting merge for [PR#868](https://github.com/rokwire/rokwire-building-blocks-api/pull/868)

**Comment by minump** [comment]
> Converting to draft PR . Awaiting merge for [PR#868](https://github.com/rokwire/rokwire-building-blocks-api/pull/868)

This issue is not fixed in PR#868. Converting to ready-for-review.

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 1]
```diff
@@ -1215,8 +1227,13 @@ <h3>Add a Talent to this Contribution</h3>
                                 <div class="form-group">
                                     <label class="control-label col-sm-2">User Privacy Level</label>
                                     <div class="col-sm-10">
-                                        <input max="5" min="1" name="talent_minUserPrivacyLevel"
-                                               step="1" type="number" value="{{ talent|filter_nested_dict(["minUserPrivacyLevel"]) }}"/>
+                                        {%  if is_editable %}
+                                            <input max="5" min="1" name="talent_minUserPrivacyLevel"
+                                                   step="1" type="number" value="{{ talent|filter_nested_dict(["minUserPrivacyLevel"]) }}"/>
+                                        {% else %}
+                                            <input max="5" min="1" name="talent_minUserPrivacyLevel"
```
Comment (ywkim312) [inline contributions/catalog/webapps/templates/contribute/contribute.html:1234]: can you put id for this one if it is in the new PR, you can check the PR https://github.com/rokwire/rokwire-building-blocks-api/pull/868

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 2]
```diff
@@ -1215,8 +1227,13 @@ <h3>Add a Talent to this Contribution</h3>
                                 <div class="form-group">
                                     <label class="control-label col-sm-2">User Privacy Level</label>
                                     <div class="col-sm-10">
-                                        <input max="5" min="1" name="talent_minUserPrivacyLevel"
-                                               step="1" type="number" value="{{ talent|filter_nested_dict(["minUserPrivacyLevel"]) }}"/>
+                                        {%  if is_editable %}
+                                            <input max="5" min="1" name="talent_minUserPrivacyLevel"
```
Comment (ywkim312) [inline contributions/catalog/webapps/templates/contribute/contribute.html:1231]: Can you check if there is id for this one in the other PR https://github.com/rokwire/rokwire-building-blocks-api/pull/868. If so, can you add the id in here?

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**